langkit/templates/pkg_rewriting_impl_body_ada.mako

```diff
@@ -715,6 +881,11 @@ package body ${ada_lib_name}.Rewriting_Implementation is
       Index  : Positive;
       Child  : Node_Rewriting_Handle) is
    begin
+      ${pre_check_nrw_handle("Handle")}
+      ${pre_check_is_list_kind("Kind (Handle)")}
+      ${pre_check_child_index("Handle", "Index", True)}
```

Comment:
Please add “for_insertion=” (so we have a name, and thus visible meaning, near this strange looking bool literal).